wire-ios-data-model/Source/Model/Message/ZMClientMessage+Encryption.swift

```diff
@@ -431,6 +431,7 @@ extension GenericMessage {
         } else if client.failedToEstablishSession {
             // If the session is corrupted, we will send a special payload.
             let data = ZMFailedToCreateEncryptedMessagePayloadString.data(using: String.Encoding.utf8)!
+            logger.error("Failed to create encryptedMessage payload string: \(data)")
```

Comment:
@johnxnguyen  Maybe it's better to say "Failed to establish the session" ?

Reply:
or "Failed to create encryptedMessage, session is not established with a client"

Reply:
updated according to above message